query/security/ldap/src/test/groovy/org/codice/ddf/admin/ldap/discover/LdapTestClaimMappingsSpec.groovy

```diff
@@ -89,7 +89,7 @@ class LdapTestClaimMappingsSpec extends Specification {
 
         // Initialize bad paths
         baseMsg = [LdapTestClaimMappings.FIELD_NAME]
-        badPaths = [badOrMissingUserNameAttrPath: baseMsg + [LdapDirectorySettingsField.USER_NAME_ATTRIBUTE],
+        badPaths = [badOrMissingUserNameAttrPath: baseMsg + [LdapDirectorySettingsField.LOGIN_USER_ATTRIBUTE],
```

Comment:
We should update this name to `badOrMissingLoginUserAttrPath`

Reply:
fixed